zivid_camera/test/test_zivid_camera.cpp

```diff
@@ -397,18 +531,19 @@ TEST_F(ZividNodeTest, testCapture2D)
     ASSERT_EQ(img.is_bigendian, false);
     ASSERT_EQ(img.data.size(), img.step * img.height);
 
-    auto verifyPixelColor = [&](const FileCameraExpectedRGB& expectedRGB) {
-      constexpr unsigned char expectedA = 255;
-      const auto index = expectedRGB.row * img.step + bytes_per_pixel * expectedRGB.col;
-      ASSERT_EQ(img.data[index], expectedRGB.r);
-      ASSERT_EQ(img.data[index + 1], expectedRGB.g);
-      ASSERT_EQ(img.data[index + 2], expectedRGB.b);
-      ASSERT_EQ(img.data[index + 3], expectedA);
-    };
-
-    for (const auto& expectedRGB : miscObjectsExpectedRGBs)
+    const auto frame2D = capture2DViaSDKDefaultSettings();
+    const auto expectedRGBA = frame2D.imageRGBA();
+    ASSERT_EQ(img.width, expectedRGBA.width());
```

Comment:
Could consider re-using checking the rgba image which is equal in 2D and 3D.
If they're different types, we can just keep as is.

Reply:
Adding helper method for this

Reply:
Fixed